plugwise/helper.py

```diff
@@ -180,12 +191,12 @@ class SmileComm:
 
     def __init__(
         self,
-        host,
-        password,
-        username,
-        port,
-        timeout,
-        websession,
+        host: str,
+        password: str,
+        username: str,
+        port: str,
```

Comment:
```suggestion
        port: int,
```

Reply:
Tnx, I must be getting tired, I didn't see that one.